python/apps/utils/rabbitmq.py

```diff
@@ -84,3 +84,73 @@ def call(self, queryset=None, serializer=None, exchange_name='', exchange_type='
         print(" [.] Got %r" % self.response)
 
         return self.response
+
+    def on_response(self, ch, method, properties, body):
+        print('I have got the messsage from server')
+        self.response = body
+
+
+
+    def call_java(self, queryset=None, serializer=None, exchange_name='', exchange_type='topic', q_receiving='',
+             q_sending='', message=''):
+
+        self.response = None
+        self.q_sending = q_sending
+        self.q_receiving = q_receiving
+
+        self.channel.queue_declare(queue=self.q_receiving, durable=True)
+        self.channel.exchange_declare(exchange=exchange_name, exchange_type=exchange_type, durable=True)
+        if queryset and serializer:
+            message = self.message_to_server(queryset, serializer)
+            cor_id = json.loads(message)
+            uuid = cor_id['uuid']
+        else:
+            message = message
```

Comment:
Did not get the meaning of this statement

Reply:
message variable is assigned to function input.